R/conttables.b.R

```diff
@@ -590,8 +592,288 @@ contTablesClass <- R6::R6Class(
 
                 othRowNo <- othRowNo + 1
             }
+            private$.runPhocTab(data=data)
+        },
+        .initPhocTab=function(data=data) {
+            rowVarName <- self$options$rows
+            colVarName <- self$options$cols
+            layerNames <- self$options$layers
+            phoc       <- self$results$phoc
+
+            subNamesPh  <- c('[resU]', '[resP]', '[resS]', '[resA]')
+            subTitlesPh <- c(.('Unstandardized'), .('Pearson'), .('Standardized'), .('Adjusted'))
+            visiblePh   <- c('(resU)', '(resP)', '(resS)', '(resA)')
+            typesPh     <- c('number', 'number', 'number', 'number')
+            formatsPh   <- c('', '', '', '')
+
+            reversed <- rev(layerNames)
+            for (i in seq_along(reversed)) {
+                layer <- reversed[[i]]
+                phoc$addColumn(name=layer, type='text', combineBelow=TRUE)
+            }
+
+            # add the row column, containing the row variable
+            # fill in dots, if no row variable specified
+
+            if ( ! is.null(rowVarName))
+                title <- rowVarName
+            else
+                title <- '.'
+
+            phoc$addColumn(
+                name=title,
+                title=title,
+                type='text')
+
+            # add the column columns (from the column variable)
+            # fill in dots, if no column variable specified
+
+            if ( ! is.null(colVarName)) {
+                superTitle <- colVarName
+                levels <- base::levels(data[[colVarName]])
```

Comment:
You don't have call functions from the `base` namespace explicitly (otherwise you'd have to `base::` every base function that you use. So this can just be:
 ```suggestion
                 levels <- levels(data[[colVarName]])
```

Reply:
I did as you suggested, but since it was the same in the other functions that I did not touch and I had followed the same calling method.